lib/dynamoid/adapter_plugin/aws_sdk_v2.rb

```diff
@@ -84,36 +84,55 @@ def client
         @client
       end
 
-      # Puts or deletes multiple items in one or more tables
+      # Puts multiple items in one tables
```

Comment:
This is not used for deletes?

`table` should be singular as you have it here.

Reply:
> This is not used for deletes?